### ISSUE TITLE
A question about the get_audio_device_ids.py file

### ISSUE THREAD

**Issue by teteterence2** [SociallyIneptWeeb/LanguageLeapAI#34]
C:\Users\Terence\Downloads\LanguageLeapAI-main\LanguageLeapAI-main\src\modules>python get_audio_device_ids.py
Traceback (most recent call last):
  File "C:\Users\Terence\Downloads\LanguageLeapAI-main\LanguageLeapAI-main\src\modules\get_audio_device_ids.py", line 1, in <module>
    import speech_recognition as sr
ModuleNotFoundError: No module named 'speech_recognition'

^
||
Hello everyone, am I supposed to run the file with command prompt?
I am new to programming so Im confused with stuffs
Any replies would be appreciated O7

**Comment by hoangvu12** [comment]
Have you installed all the dependencies?

**Comment by teteterence2** [comment]
Im not sure, I just need to download the file in the main page right? I downloaded it and unzipped it already

**Comment by SociallyIneptWeeb** [comment]
Read the documentation at docs/INSTALLATION.md. also try following the entire setup documentation first before running anything.

**Comment by teteterence2** [comment]
Ohh i found it. Now I ran the code pip install -r requirements.txt
and it seem worked but theres some red words at the end

ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
spyder 5.2.2 requires pyqt5<5.13, which is not installed.
spyder 5.2.2 requires pyqtwebengine<5.13, which is not installed.
daal4py 2021.6.0 requires daal==2021.4.0, which is not installed.
anaconda-project 0.11.1 requires ruamel-yaml, which is not installed.
numba 0.55.1 requires numpy<1.22,>=1.18, but you have numpy 1.24.2 which is incompatible.
conda-repo-cli 1.0.20 requires clyent==1.2.1, but you have clyent 1.2.2 which is incompatible.
conda-repo-cli 1.0.20 requires nbformat==5.4.0, but you have nbformat 5.5.0 which is incompatible.
conda-repo-cli 1.0.20 requires requests==2.28.1, but you have requests 2.28.2 which is incompatible.

the first red code


I ignored them and continued until running the voicevox notebook file on google colab, but then i got some other red codes at the end of the dependencies part too. What do i need to do? Is it ok leave it and continue?

ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
ipython 7.9.0 requires jedi>=0.10, which is not installed.
cmdstanpy 1.1.0 requires numpy>=1.21, but you have numpy 1.20.0 which is incompatible.

the second red code
